src/SFA.DAS.ApprenticePortal/Models/Apprenticeship.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticePortal.Models
+{
+    public class Apprenticeship
+    {
+        public long Id { get; set; }
+        public Guid ApprenticeId { get; set; }
+        public long CommitmentsApprenticeshipId { get; set; }
+        public string EmployerName { get; set; }
+        public long EmployerAccountLegalEntityId { get; set; }
+        public long TrainingProviderId { get; set; }
+        public string TrainingProviderName { get; set; }
+        public bool? TrainingProviderCorrect { get; set; }
+        public bool? EmployerCorrect { get; set; }
+        public RolesAndResponsibilitiesConfirmations RolesAndResponsibilitiesConfirmations { get; set; }
+        public bool? ApprenticeshipDetailsCorrect { get; set; }
+        public bool? HowApprenticeshipDeliveredCorrect { get; set; }
+        public string CourseName { get; set; }
+        public int CourseLevel { get; set; }
+        public string CourseOption { get; set; }
+        public int CourseDuration { get; set; }
+        public DateTime PlannedStartDate { get; set; }
+        public DateTime PlannedEndDate { get; set; }
+        public DateTime ConfirmBefore { get; set; }
+        public DateTime? ConfirmedOn { get; set; }
+        public long RevisionId { get; set; }
+        public ChangeOfCircumstanceNotifications ChangeOfCircumstanceNotifications { get; set; }
+        public DateTime ApprovedOn { get; set; }
+        public DateTime? LastViewed { get; set; }
+    }
```

Comment:
We appear to be returning more details here than we need. If we are not using fields there is a very good argument to say they should be present. It becomes a little bit of a maintenance problem. Ie. if ChangeOfCircumstanceNotifications is updated do we have to update it here (if our consuming web site has no interest in it)?

Reply:
reduced dtos to required columns only